README.md

```diff
@@ -195,11 +209,19 @@ end
 
 ### Calculate HMAC
 
+#### Using API secret from global configuration
 ```
 user_email = "joe@example.com"
 hmac = MagicBell.hmac(user_email)
```

Comment:
Should we instead calculate the HMAC on the client object? (and keep this for backward compatibility but deprecate it)?

Reply:
@ousmanedev

Reply:
@unamashana 

I moved the `hmac` calculation logic to `MagicBell::Client`. Can you also have a look? @josuemontano